assets/js/base/components/price-slider/utils.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * External dependencies
+ */
+import { NumberFormatValues } from 'react-number-format';
+
+export const withMaxValueLimit = ( {
+	maxConstraint,
+	minorUnit,
+}: {
+	maxConstraint: number;
+	minorUnit: number;
+} ) => ( { floatValue }: NumberFormatValues ): boolean => {
+	const maxPrice = maxConstraint / 10 ** minorUnit;
+
+	return floatValue !== undefined && floatValue <= maxPrice;
+};
+
+export const withMinValueLimit = ( {
```

Comment:
A doc block explaining the goal of this function would be great.
`with` prefix is usually used for HOCs, I find it rather confusing here. The logic in the return is also confusing from a first look and probably deserves to be dumped down or explained.

Reply:
```suggestion
// Check if that the value is greater than the lowest available price and minor than the current max price.
export const checkMinValueLimit = ( {
```

What do you think?

Reply:
Yes! Much better, would also add a comment to `withMaxValueLimit`.